README.md

```diff
@@ -205,6 +211,19 @@ The Diffie-Hellman group should have at least 1024 bit and can be created with t
 ```
 # openssl dhparam -out /etc/exim4/dhparam.pem 2048
 ```
+In the transport section, you need to modify the user and group to the exim-user `system_aliases`-transport (around line 800):
+```
+system_aliases:
+  driver = redirect
+  allow_fail
+  allow_defer
+  data = ${lookup{$local_part}lsearch{/etc/aliases}}
+  user = <exim-user>
+  group = <exim-group>
+  file_transport = address_file
+  pipe_transport = address_pipe
```

Comment:
So you get mails delivered into a file of a local user? The debian config says:
````
# This router handles aliasing using a traditional /etc/aliases file.
# Delivery to arbitrary directories, files, and piping to programs in
# /etc/aliases is disabled per default.
````
All these conditional parts make the configuration file hard to read. You need to add to the documentation that if they want to use it, they have to define variables `SYSTEM_ALIAS_*` and where to put it.

Probably, most users only need the `/etc/alias` readout and we can comment the rest out. But this must be done in the configuration file.